closed/src/java.base/share/classes/jdk/crypto/jniprovider/NativeCrypto.java

```diff
@@ -202,6 +251,8 @@ public void run() {
 
     private static final native long loadCrypto(boolean trace);
 
+    private static final native boolean isOpenSSLFIPS();
```

Comment:
Suggest adding this after `isMD5Available()`.

Reply:
Moved.